src/applications/facility-locator/tests/components/ProviderTraining.unit.spec.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { expect } from 'chai';
+import { render } from '@testing-library/react';
+import CCProviderResult from '../../components/search-results-items/CCProviderResult';
+import testData from '../../constants/mock-facility-data-v2.json';
+
+describe('CCProviderResult With Training', () => {
+  it('Should render CCProviderResult, serviceType Podiatrist without training', async () => {
```

Comment:
Consider adding some line breaks between `it` statements for readability here

Reply:
added